docs/en/observability/synthetics-reference/lightweight-config/tcp.asciidoc

```diff
@@ -0,0 +1,126 @@
+:hardbreaks-option:
+
+[%header]
+|===
+| Option (type) | Description
+
+// hosts
+| [[monitor-tcp-hosts]] *`hosts`*
+(list of <<synthetics-lightweight-data-string,string>>s)
+a| *Required*. A list of hosts to ping. The entries in the list can be:
+
+* *A plain host name, such as `localhost`, or an IP address.*
+  If you specify this option, you must also specify a value for <<monitor-tcp-ports,`ports`>>.  If the monitor is {heartbeat-ref}/configuration-ssl.html[configured to use SSL], Heartbeat establishes an SSL/TLS-based connection. Otherwise, it establishes a plain TCP connection.
+
+* *A hostname and port, such as `localhost:12345`.*
+  Heartbeat connects to the port on the specified host. If the monitor is {heartbeat-ref}/configuration-ssl.html[configured to use SSL], Heartbeat establishes an SSL/TLS-based connection. Otherwise, it establishes a TCP connection.
+
+* *A full URL using the syntax `scheme://<host>:[port]`*, where:
+** `scheme` is one of `tcp`, `plain`, `ssl` or `tls`. If `tcp` or `plain` is specified, Heartbeat establishes a TCP connection even if the monitor is configured to use SSL. If `tls` or `ssl` is specified, Heartbeat establishes an SSL connection. However, if the monitor is not configured to use SSL, the system defaults are used (currently not supported on Windows).
+** `host` is the hostname.
+** `port` is the port number. If `port` is missing in the URL, the <<monitor-tcp-ports,`ports`>> setting is required.
```

Comment:
Preview: [`hosts`](https://observability-docs_2729.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-lightweight.html#monitor-tcp-hosts)

Should the mention of "Heartbeat" here be removed? If yes, what's the right alternative? Are all these options (copied from the Heartbeat docs) relevant to projects?

Reply:
++, replaced with "Synthetics" as mentioned above